packages/tatum-core/src/model/request/GetOrderBook.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNotEmpty, IsNumber, IsString, Length, Matches, Min } from 'class-validator'
+import { TimeFrame } from '.'
+
+export class GetOrderBook {
+  @IsNotEmpty()
+  @IsString()
+  @Matches(/^[A-Za-z0-9_-]+\/[A-Za-z0-9_-]+$/)
+  @Length(3, 30)
+  public pair: string
+
+  @IsNotEmpty()
+  @IsNumber()
+  @Min(0)
+  public from: number
+
+  @IsNotEmpty()
+  @IsNumber()
+  @Min(0)
+  public to: number
+
+  @IsNotEmpty()
+  @IsString()
```

Comment:
Can we use `@IsEnum(entity: object)`? See https://github.com/typestack/class-validator#validation-decorators

Reply:
yeah, I missed that somehow, will update!